packages/code-du-travail-frontend/src/outils/DureePreavisLicenciement/steps/Result.js

```diff
@@ -0,0 +1,123 @@
+import React from "react";
+import PropTypes from "prop-types";
+import data from "@cdt/data...preavis-licenciement/data.json";
+
+import { SectionTitle, Highlight } from "../../common/stepStyles";
+
+import {
+  filterSituations,
+  getSituationsFor,
+  recapSituation,
+  getRef
+} from "../../common/situations.utils";
+
+const { situations: allSituations } = data;
+
+function StepResult({ form }) {
+  const { values } = form.getState();
+  const { ccn, cdt, seriousMisconduct, disabledWorker, criteria = {} } = values;
+  const idcc = ccn ? ccn.num : "0000";
+
+  // Situation CDT
+  const initialCDTSituations = getSituationsFor(allSituations, {
+    idcc: "0000"
+  });
+  const possibleCDTSituations = filterSituations(initialCDTSituations, cdt);
+
+  const [situationCDT] = possibleCDTSituations;
+  const {
+    criteria: { ancienneté: seniorityCDT, ...situationCDTCriteria },
+    duration: durationCDT
+  } = situationCDT;
+
+  // Situation CC
+  const initialCCSituations = getSituationsFor(allSituations, { idcc });
+  const possibleCCSituations = filterSituations(initialCCSituations, criteria);
+
+  const [situationCC] = possibleCCSituations;
+  const {
+    criteria: { ancienneté: seniorityCC, ...situationCCCriteria },
+    duration: durationCC
+  } = situationCC || { criteria: {} };
+
+  return (
+    <>
+      <SectionTitle>Durée du préavis</SectionTitle>
+      <p>
+        À partir des éléments que vous avez saisis, la durée du préavis de
+        licenciement est estimée à&nbsp;:
+        <p>
```

Comment:
I not sure it's ok to nest paragraph inside paragraph

Reply:
Clearly!